src/components/demos/read/Demo.js

```diff
@@ -155,7 +154,7 @@ export default function ReadDemo(props) {
                       InputErrorWarning={InputErrorWarning}
                     />
                     <Button
-                      {...readDemo.theme.form.saveSecretButton}
+                      {...state.theme.form.savePasswordButton}
```

Comment:
the button styling is broken 
<img width="780" alt="image" src="https://user-images.githubusercontent.com/40722025/192237152-e32d577b-49e0-4b2d-a10a-ac60cde595cf.png">

Reply:
this has been fixed! Used password instead Secretl.